src/pudl/glue/ferc1_eia.py

```diff
@@ -179,7 +183,7 @@ class GenericPlantFerc1TableTransformer(Ferc1AbstractTableTransformer):
     Intended for use in compiling all plant names for manual ID mapping.
     """
 
-    def __init__(self, table_id: Ferc1TableId, **kwargs):
+    def __init__(self, table_id: SourceFerc1, **kwargs):
```

Comment:
Did you really mean to swap the table ID for the source here? They seem like two distinct things.

Reply:
good catch! that was a mega find and replace moment... not sure how this one went so wrong